internal/serve/middleware/middleware_test.go

```diff
@@ -151,6 +154,7 @@ func TestRecoverHandler(t *testing.T) {
 	req, err := http.NewRequest("GET", "/", nil)
 	require.NoError(t, err)
 	rr := httptest.NewRecorder()
+	appTrackerMock.On("CaptureException", mock.Anything).Return(nil)
```

Comment:
The same suggestion that I've made on the `InternalServerError`, can we test the error value here?

Reply:
Done